subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -12,6 +12,31 @@ config BT_MESH_NRF_MODELS
 	  Common mesh model support modules, required by all Nordic BT mesh
 	  models.
 
+menuconfig BT_MESH_ACKNOWLEDGED_MESSAGE
+	bool "Configuration of the waiting time for acknowledged messages"
+	default y
+	select BT_MESH_NRF_MODELS
+	help
+	  The Bluetooth Mesh Profile recommends setting this timeout to
+	  a minimum of 30 seconds. However, the exact value is application specific.
+	  The final timeout is the sum of the per hop timeout multiplied
+	  on the number of hops and the base timeout
```

Comment:
No need for this entry. There's no valid scenario for disabling it, as that would remove the defines for the numbers and fail building. 

If you want to collect these under a specific menu, you can use 
`menu` and `endmenu`, like we do for the property models:
https://github.com/nrfconnect/sdk-nrf/blob/559947f1bacc17fcf11b14a12bde52c2f3c12c89/subsys/bluetooth/mesh/Kconfig.models#L207-L228

Reply:
Yep, I'd like to keep the menu help description to give to customers the glue about spec requirement and formula.